scripts/qa_crowbarsetup.sh

```diff
@@ -1609,18 +1609,59 @@ function set_proposalvars()
     # Determine if we went through an upgrade
     [ -f /etc/cloudsource ] && export cloudsource=$(</etc/cloudsource)
 
-    wantswift=1
-    [ -z "$want_swift" ] && wantceph=1
-    [[ -n "$wanthyperv" ]] && wantswift= && wantceph= && networkingmode=vlan
+    ### dynamic defaults
+    case "$nodenumber" in
+        0|1)
+            wantswift=
+            wantceph=
+        ;;
+        2)
+            wantswift=1
+            wantceph=
+        ;;
+        *)
+            wantswift=
+            wantceph=1
+        ;;
+    esac
+
+    ### filter (temporarily changing defaults)
+    # F1: hyperV only without swift and ceph
+    [ -n "$wanthyperv" ] && wantswift= && wantceph= && networkingmode=vlan
+
+    ### user requests (can override defaults and filters)
```

Comment:
How can this override work?  It seems that setting the defaults above will eliminate any user requests.